tests/testthat/test-workflow-bbi.R

```diff
@@ -305,5 +305,160 @@ withr::with_options(list(
     run_times <- model_summaries(mods) %>% check_run_times(.wait = FALSE) %>% suppressWarnings()
     expect_equal(dim(run_times), c(3, 3))
   })
+
+  describe("run_nmtran", {
+    it("nmtran_setup: executable and nonmem version", {
+      mod1 <- read_model(file.path(MODEL_DIR_BBI, "1"))
+      # Using model object, looks for bbi.yaml
+      nmtran_specs <- nmtran_setup(mod1)
+      # Confirm executable
+      expect_equal(nmtran_specs$nmtran_exe, "/opt/NONMEM/nm74gf/tr/NMTRAN.exe")
+      # Confirm NONMEM version
+      expect_equal(nmtran_specs$nonmem_version, "nm74gf")
+
+      # Passed nonmem version
+      nmtran_specs <- nmtran_setup(mod1, .nonmem_version = "nm75")
+      # Confirm executable
+      expect_equal(nmtran_specs$nmtran_exe, "/opt/NONMEM/nm75/tr/NMTRAN.exe")
+      # Confirm NONMEM version
+      expect_equal(nmtran_specs$nonmem_version, "nm75")
+
+      # Passed config_path
+      nmtran_specs <- nmtran_setup(.config_path = file.path(MODEL_DIR_BBI, "bbi.yaml"))
+      # Confirm executable
+      expect_equal(nmtran_specs$nmtran_exe, "/opt/NONMEM/nm74gf/tr/NMTRAN.exe")
+      # Confirm NONMEM version
+      expect_equal(nmtran_specs$nonmem_version, "nm74gf")
+
+      # Incorrect nonmem version
+      expect_error(
+        nmtran_setup(mod1, .nonmem_version = "nm74"),
+        "Must specify a valid `.nonmem_version`"
+      )
+
+      # no configuration file found
+      expect_error(
+        nmtran_setup(mod1, .config_path = file.path(tempdir(), "bbi.yaml")),
```

Comment:
`tempdir()` returns the session directory and isn't exclusively under this test's control. `tempfile()` would be safer.